src/app/screens/ordinalDetail/useOrdinalDetail.ts

```diff
@@ -23,8 +22,7 @@ export default function useOrdinalDetail() {
   const { ordinalsAddress, network, selectedAccount, hasActivatedRareSatsKey } =
     useWalletSelector();
   const { id } = useParams();
-  const { selectedOrdinal } = useNftDataSelector();
-  const { data: ordinalData, isLoading } = useAddressInscription(id!, selectedOrdinal);
+  const { data: ordinalData, isLoading } = useAddressInscription(id!);
```

Comment:
The ID is always populated as it's part of the route. In the final proper fix, we can also remove the second argument for useAddressInscription as it would just get the data from the fetch cache.